transforms/no-vars.js

```diff
@@ -0,0 +1,79 @@
+/** 
+ * Transform all your `var`s to `let` and `const`.
+ *
+ * The only catch is it may not catch nested bodies that mutate anything beyond
```

Comment:
what’s the best way to climb back up parent scopes to detect previous declarations and potential mutations?